apps/ui/src/components/sidebar/utils.tsx

```diff
@@ -20,43 +21,45 @@ interface SideBar {
   component: JSX.Element;
 }
 
-export const iconsMapper = (
-  routes: string[] | undefined,
-): Record<
+export const iconsMapper = (): Record<
```

Comment:
turn this into regular object `export const iconsMapper: Record<string,  {
    active: React.ReactNode;
    inactive: React.ReactNode;
    hidden?: boolean;
    type: string;
    id: string;
  }> = {...}`

Reply:
Good idea, now we dont pass in the param we dont need the function.